bin/ovpn_genconfig

```diff
@@ -160,7 +160,7 @@ OVPN_DNS=1
 OVPN_DEVICE="tun"
 OVPN_DEVICEN=0
 OVPN_KEEPALIVE="10 60"
-OVPN_DNS_SERVERS=("8.8.8.8" "8.8.4.4")
+OVPN_DNS_SERVERS=($(awk '/nameserver/{print $2}' /etc/resolv.conf))
```

Comment:
sorry for a newbie question, this `/etc/resolv.conf` is from the docker image, or from the host system?